### PR TITLE
Rename the Swift binary

### PR DESCRIPTION
**Pull request by albinekb**
This PR changes the main binary to `aperture` so that you know what process this is in your acitivity monitor, currently it was just "main", which is a bit confusing 😄 

See here for an example (before):
kapture 2017-05-30 at 2 55 54

~Also, seems like the processes aren't exited correctly? 🤔  when playing around, i had 10's of `main` processes in my activity monitor 😕~

see #35 👍

### CODE REVIEW

**Comment by albinekb** [comment]
I tried using the [child_process `argv0`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) option, that made it show up correctly in htop, but not in Activity Monitor.

**Comment by matheuss** [comment]
As far as I recall, the `main` is mandatory 🤔 I think it's for when you run it as a script (which I used to do when developing). Will download Xcode etc and test it 👍

**Comment by albinekb** [comment]
another option is to symlink `main` to `aperture` 🤔

File: swift/aperture/main.swift [hunk 1]
```diff
@@ -55,10 +55,11 @@ func record() throws {
...
 func usage() {
-  print("usage: main <list-audio-devices | <destinationPath> <fps> <crop-rect-coordinates> <show-cursor> <highlight-clicks> <display-id> <audio-device-id>>")
-  print("examples: main ./file.mp4 30 0:0:100:100 true false 1846519 'AppleHDAEngineInput:1B,0,1,0:1'")
-  print("          main ./file.mp4 30 none true false main none")
-  print("          main list-audio-devices")
+  let name = CommandLine.arguments.first?.components(separatedBy: "/").last ?? "main"
+  print("usage: \(name) <list-audio-devices | <destinationPath> <fps> <crop-rect-coordinates> <show-cursor> <highlight-clicks> <display-id> <audio-device-id>>")
+  print("examples: \(name) ./file.mp4 30 0:0:100:100 true false 1846519 'AppleHDAEngineInput:1B,0,1,0:1'")
+  print("          \(name) ./file.mp4 30 none true false main none")
+  print("          \(name) list-audio-devices")
```
Comment (sindresorhus) [inline swift/aperture/main.swift]: Just hardcode the name.

File: package.json [hunk 2]
```diff
@@ -17,12 +17,12 @@
   },
   "scripts": {
     "test": "xo",
-    "build": "cd swift && xcodebuild archive -derivedDataPath $(mktemp -d) -scheme aperture",
+    "build": "cd swift && xcodebuild archive -derivedDataPath $(mktemp -d) -scheme aperture && mv main ../aperture",
```
Comment (sindresorhus) [inline package.json]: Can you change this in the Xcode "Run Script" instead? That way we can use Xcode to build too. Good when developing.

**Comment by sindresorhus** [comment]
> As far as I recall, the main is mandatory 

I don't think it is.

**Pull request merged**